source/codegen/metadata/nisync/functions.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+functions = {
+    "init": {
```

Comment:
You should mark all init functions with `'init_method': True,` like [niscope's Init method](https://github.com/ni/grpc-device/blob/main/source/codegen/metadata/niscope/functions.py#L2120). This will make it generate the correct method body to register the new session in the `session_repository` so clients can interact with it through the `session_service` ([corresponding proto file](https://github.com/ni/grpc-device/blob/main/source/protobuf/session.proto)).

Reply:
added `'init_method: True,'` to functions.py